### DISCUSSION TITLE
How to change chain logo

### DISCUSSION THREAD

**Discussion by amcs99** [WalletConnect/walletconnect-monorepo#3691] [Developer Support] [answered]
Hi, I'm using web3modal and wagmi together. My dapp is aimed to support multiple chains including [Base.](https://base.org/) But when the modal is opened the logo of Base is shown as a testnet chain. I just want to display the official logo of Base. How to do this? Thanks

**Comment by glitch-txs** [comment, marked as answer]
I'm merging [this](https://github.com/WalletConnect/walletconnect-docs/pull/1002) next week 🤫 but we added this in Web3Modal v3

You can see the preview of the PR [here](https://walletconnect-docs-git-feat-w3m-structure-walletconnect1.vercel.app/2.0/web3modal/react/options#chainimages)
